qubesmanager/tests/test_backup_01.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
+#
+# The Qubes OS Project, https://www.qubes-os.org/
+#
+# Copyright (C) 2016 Marek Marczykowski-Górecki
+#                                       <marmarek@invisiblethingslab.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+import logging.handlers
+import sys
+import unittest
+import unittest.mock
+
+from PyQt4 import QtGui, QtTest, QtCore
+from qubesadmin import Qubes
+import qubesmanager.backup as backup_gui
+
+
+class BackupTest(unittest.TestCase):
+    def setUp(self):
+        super(BackupTest, self).setUp()
+
+        # mock up nonexistence of saved backup settings
+        self.patcher = unittest.mock.patch('builtins.open')
+        self.mock_open = self.patcher.start()
+        self.mock_open.side_effect = FileNotFoundError()
+        self.addCleanup(self.patcher.stop)
+
+        self.qapp = Qubes()
+        self.qtapp = QtGui.QApplication(sys.argv)
+        self.dialog = backup_gui.BackupVMsWindow(self.qtapp, self.qapp)
+
+    def tearDown(self):
+        del self.dialog
+        del self.qtapp
+        del self.qapp
+        super(BackupTest, self).tearDown()
+
+    def test_window_loads(self):
```

Comment:
Since tests order can be very helpful for interpreting results, better number them, as you've done in other files.